compiler/pkg/parser/pkg.go

```diff
@@ -722,3 +737,62 @@ func (p *Package) ValueSpecToString(t *ast.ValueSpec) (string, error) {
 	}
 	return buf.String(), nil
 }
+
+func IsNexusGraphqlNullField(f *ast.Field) bool {
+	if f == nil {
+		return false
+	}
+
+	if f.Tag != nil {
+		tags := ParseFieldTags(f.Tag.Value)
+		if val, err := tags.Get(string(GRAPHQL_NULLABLE_ANNOTATION)); err == nil {
+			if strings.ToLower(val.Name) == "false" {
+				return false
+			}
+		}
+	}
+
+	return true
```

Comment:
Will `GRAPHQL_NULLABLE_ANNOTATION` be set to true in true scenario? If yes, then this return true should come inside `if f.Tag != nil`, is it not?